samples/dlp_inspect_image_file.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', filepath = 'path/to/image.png') {
+  // [START dlp_inspect_image_file]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const filepath = 'path/to/image.png';
+
+  // Imports the Google Cloud Data Loss Prevention library
+  const dlp = require('@google-cloud/dlp');
+
+  // Import other required libraries
+  const fs = require('fs');
+  const util = require('util');
+  const readFile = util.promisify(fs.readFile);
+
+  // Instantiates a client
+  const client = new dlp.DlpServiceClient();
+
+  async function inspectImageFile() {
+    // Get the bytes of the file
+    const fileBytes = Buffer.from(await readFile(filepath)).toString('base64');
+
+    // Construct request
+    const request = {
+      parent: client.projectPath(projectId),
+      item: {
+        byteItem: {
+          type: 'IMAGE',
+          data: fileBytes,
+        },
+      },
+      inspectConfig: {
+        // The infoTypes of information to match
+        infoTypes: [
+          {name: 'PHONE_NUMBER'},
+          {name: 'EMAIL_ADDRESS'},
+          {name: 'CREDIT_CARD_NUMBER'},
+        ],
+        // Whether to include the matching string
+        includeQuote: true,
+      },
+    };
+
+    // Run request
+    const [response] = await client.inspectContent(request);
+    const findings = response.result.findings;
+    console.log(`Findings: ${!findings.length ? 'None' : ''}`);
+    for (const finding of findings) {
+      console.log(`  Quote: ${finding.quote}`);
```

Comment:
As a general rule, should every inner-loop indent its print statements 2 spaces?

Is it important to indent at all?

Reply:
Hmm good question, I'm trying to find a format that groups the results being looped over. What do you think about

```js
console.log(`Findings: ${!findings.length ? 'None' : ''}`);
for (const finding of findings) {
  console.log(`- Quote: ${finding.quote}`);
  console.log(`  Info type: ${finding.infoType.name}`);
  console.log(`  Likelihood: ${finding.likelihood}`);
}
```

which would produce:

```
Findings: 2
- Quote: something something
  Info type: something
  Likelihood: 0.7
- Quote: something something
  Info type: something
  Likelihood: 0.2
```

Reply:
I prefer without the `-`, as a reader I don't necessarily understand why Quote is different from the others.

> **"Our focus is on cloud.google.com"**

If people want to add space indentation, that's OK 🤷🏼‍♀️
 - Personally, I'm against it altogether.
 - On the website, the visualization of the indentation is implicit *(the source code itself is indented)*
 - As a developer running our samples, I do not have the expectation that they output with pretty indentation. as a developer, I may be accustomed to using the `gcloud` command-line tool. `gcloud` does not add much pretty indentation.
 - Sidenote: I would like to outlaw ``console.log(`\tQuote: ${finding.quote}`)``. Eyes can see `  Quote` just fine but I find `\tQuote` to provide quiet poor readability in comparison

----
----

I just don't see the advantage of changing the code embedded in the website to change this:

```
Company: projects/rebecca-gcp/companies/086623f6-d9f3-4d62-95ca-3730e2d68d0c
Display name: Second company
External ID: 45678-identifier
Company: projects/rebecca-gcp/companies/099e996a-b315-44d2-bc21-426fe99d132d
Display name: First company
External ID: 123456-identifier
```

to this:

```
Company: projects/rebecca-gcp/companies/086623f6-d9f3-4d62-95ca-3730e2d68d0c
  Display name: Second company
  External ID: 45678-identifier
Company: projects/rebecca-gcp/companies/099e996a-b315-44d2-bc21-426fe99d132d
  Display name: First company
  External ID: 123456-identifier
```

I always write my samples like so:

```js
for (const finding of findings) {
  console.log(`Quote: ${finding.quote}`);
  console.log(`Info type: ${finding.infoType.name}`);
  console.log(`Likelihood: ${finding.likelihood}`);
}
```

**$0.25**

Reply:
Indentation removed.